server/netop_handler.go

```diff
@@ -73,22 +73,42 @@ func BootstrapNetOpPod() error {
 	return nil
 }
 
+func TcCmdError(tcCmd string, err error, cmdOut string) string {
```

Comment:
if we are not using these util functions outside, let's not expose them.

Reply:
Sure Gourish

Reply:
Except TcCmdError rest I made them not accessible outside please review again